env.sh

```diff
@@ -0,0 +1,76 @@
+cd $HOME/anthos-workshop
```

Comment:
Are we sure this will always be anthos-workshop?

Reply:
Right now the clone instructions will make it always be anthos-workshop.
We can change it to be cd ${BASE_DIR} to ensure that should instructions change to clone into different dir this will still work.